v3/proteus67/proteus67.json

```
@@ -0,0 +1,155 @@
+{
+   "name":"Proteus67",
+   "vendorId":"0x7072",
+   "productId":"0x6F64",
+   "menus": ["qmk_rgblight"],
```

```suggestion
	"keycodes": [ "qmk_lighting" ],
   "menus": ["qmk_rgblight"],
```

Missing the keycodes option

---

added keycodes option as suggested.